activerecord/lib/active_record/test_databases.rb

```diff
@@ -8,8 +8,8 @@ module TestDatabases # :nodoc:
       create_and_migrate(i, spec_name: Rails.env)
     end
 
-    ActiveSupport::Testing::Parallelization.run_cleanup_hook do |i|
-      drop(i, spec_name: Rails.env)
+    ActiveSupport::Testing::Parallelization.run_cleanup_hook do |_|
+      drop(spec_name: Rails.env)
     end
 
     def self.create_and_migrate(i, spec_name:)
```

Comment:
Let's also rename this method to `create_and_load` since we're no longer migrating.

Reply:
Got it! Thanks.